features/step_definitions/pod.rb

```diff
@@ -249,6 +249,18 @@
   @result  = res_cache
 end
 
+Given /^I collect the deployment log for pod "(.+)" until it becomes :([^\s]*?)$/ do |name, status|
+  opts = {resource_name: name}
+  timeout = timeout ? Integer(timeout) : 15 * 60   # just put a timeout so we don't hang there indefintely
+  podstatus = pod(name).wait_till_status(status.to_sym, user, timeout)
+  unless podstatus[:success]
+    logger.error(podstatu[:response])
+    raise "#{pod.name} pod was never ready"
```

Comment:
Should we say pod was never ready or pod never succeeded as in the above test script i see that we are already checking for pod ready status before collecting the logs, thanks !!

Reply:
@kasturinarra  "succeeded" just a parametes, we could also input the "running"  or "failed" status. then also confused for debug .

Reply:
okay !!

Reply:
raise "pod #{name} didn't become #{status}"

Reply:
Any update for this ?  By the way, found an typo here `podstatu`